packages/components/spinner/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
spinner should be also added to `tsconfig.all.json` file in `references` array:

`{ "path": "./packages/components/spinner" },`